app/src/gecko/java/org/mozilla/tv/firefox/ext/EngineView.kt

```diff
@@ -177,6 +180,10 @@ object WebHistory {
 val EngineView.backForwardList: WebHistory
     get() = WebHistory
 
+fun EngineView.maybeGoBackBeforeFxaSignIn() {
+    logger.warn("maybeGoBackBeforeFxaSignIn not implemented")
```

Comment:
I would replace this with a crash.  If we ever move to GV, we'll want to know about this sooner rather than later.  We can always comment it out at that time if it's considered low priority.

Reply:
I was on the fence about this so sounds good to me.